tests/integration/security/main_test.go

```diff
@@ -48,3 +50,11 @@ func TestMain(m *testing.M) {
 		}).
 		Run()
 }
+
+func setupConfig(cfg *istio.Config) {
+	if cfg == nil {
+		return
+	}
+	isMtlsEnabled = cfg.IsMtlsEnabled()
```

Comment:
This could change some behavior in the existing test? May be it's better that you apply the policy to enable mTLS for your test cases.

Reply:
It won't change any behavior in the existing test since existing tests do not use this, and in fact all rbac tests right now are not testing with mtls enabled, which is bad. I'll send another PR to have mtls enabled with rbac tests, since it's the common use case.